### ISSUE TITLE
Animation Editor not optimized for mobile browsers

### ISSUE THREAD

**Issue by Gregory-Kitching** [lix229/UnityRuntimeImport#43]
The design of the website breaks when used on a mobile device (or at least in Device Mode in Google Chrome). Specifically, the "Choose File" and "Export" buttons are blocked by the white side bar.

![Screenshot (47)](https://user-images.githubusercontent.com/79662360/110250479-66dc8400-7f49-11eb-928c-77d987cbaae4.png)

**Comment by lix229** [comment]
Duplicate of #5
